src/app/client/js/patterns/client.jsx

```diff
@@ -49,7 +49,7 @@ export let client = {
                     // Finalize the transaction
                     onApprove: function(data, actions) {
                         return actions.order.capture().then(function(details) {
-                            // Show a success message to the buyer
+                            // Show a success message to the buyer (try to avoid alerts in production environments)
```

Comment:
Can you multi-line this?

```
// Show a success message to the buyer
// NOTE: It's a best practice to remove native alert()s in production. Instead, this error should be handled or logged to your systems.
```

Sheila, what do you think? ^^

Reply:
@mstuart I have updated this comment based on @PrestonPHX suggestions.